docs/docs/en/guide/project/workflow-instance.md

```diff
@@ -40,7 +40,7 @@ Click Project Management -> Workflow -> Workflow Instance, enter the workflow in
 
 ![workflow-instance](/img/new_ui/dev/project/workflow-instance.png)
 
-- **Edit：** only can edit terminated processes. Click the "Edit" button or the workflow instance name to enter the DAG edit page. After the edit, click the "Save" button to confirm, as shown in the figure below. In the pop-up box, check "Whether to update to workflow definition" to update the workflow definition; if not checked, no effect on the workflow definition
+- **Edit：** only can edit terminated processes. Click the "Edit" button or the workflow instance name to enter the DAG edit page. After the edit, click the "Save" button to confirm, as shown in the figure below. In the pop-up box, check "Whether to upload to workflow definition" to update instance modified information to workflow definition; if not checked, no effect on the workflow definition
```

Comment:
Only workflow instance state in ```successed/failed/killed``` can be edited.

Reply:
ok, I modified the corresponding content